Makefile

```diff
@@ -0,0 +1 @@
+include mk/lint.mk
```

Comment:
one minor bikeshed: I like to be able to do `make lint` - could you add that as a make target :-). Thanks!